src/minirest.appup.src

```diff
@@ -1,5 +1,5 @@
 %%-*-: erlang -*-
-{"0.3.3",
+{"0.3.4",
```

Comment:
I'll change the `appup.src` when we have a new tag, and there're probably some instructions for reloading modules then. You could keep this file unchanged for now.